soh/src/code/z_demo.c

```diff
@@ -502,8 +502,11 @@ void Cutscene_Command_Terminator(GlobalContext* globalCtx, CutsceneContext* csCt
     }
 
     if ((csCtx->frames == cmd->startFrame) || (temp != 0) ||
+        // Skip specific cutscene in rando automatically
+        (gSaveContext.n64ddFlag && cmd->base == 33 && (csCtx->frames > 20)) ||
```

Comment:
could you explain what `csCtx->frames > 20` is doing?

what happens if you remove it?

Reply:
also probably would be good to mention which cutscene, 33 is kind of a magic number at the moment